index.ts

```diff
@@ -0,0 +1,29 @@
+import express from "express";
+import sequelize from "./src/configs/db-config";
+import Meetings from "./src/models/meetings";
+
+const app = express();
+const Meeting = new Meetings();
+
+sequelize
```

Comment:
ca si la ceilalti 2 colegi - incercati sa gasiti alta solutie, mai practica..,, eventual incercati sa vedeti in documentatia de sequelize : https://sequelize.org/docs/v6/core-concepts/model-basics/